.github/scripts/extract_source_strings.py

```diff
@@ -14,9 +14,9 @@
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        "--vpn",
+        "--input",
         required=True,
-        dest="vpn_path",
+        dest="input_path",
         help="Path to VPN project repository",
```

Comment:
This is not correct anymore, this is the `Path to the .ts file to import strings from`.